README.md

```diff
@@ -1 +1,163 @@
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Catalog of my things] <a name="about-project"></a>
+
+**[Catalog of my things]** This repo holds a command line interface app built with Ruby that will help users to keep a record of different types of things, like books, music albums, movies, and games. The app is based on the UML class diagram provided in the README and includes a database schema analogical to the class structuture of the app.
+
+## 🛠 Built With <a name="built-with"></a>
+**Ruby** : A dynamic, open source programming language with a focus on simplicity and productivity. It has an elegant syntax that is natural to read and easy to write.
+**SQL**: SQL stands for Structured Query Language · SQL lets you access and manipulate databases.
+**GitHub Flow**: GitHub flow is a lightweight, branch-based workflow. 
+
+# UML DIAGRAM
+
+![UML DIAGRAM](https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/images/catalog_of_my_things.png?raw=true)
+
+<!-- Features -->
+### Tech Stack <a name="tech-stack"></a>
+1- **Ruby**
+2- **Postgresql**
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[Add item to Database]**
+- **[List items from database]**
+- **[Save item in json files]**
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+A code editor like Visual Studio Code with Git and Ruby.
+You can check if Git is installed by running the following command in the terminal:
+
+``$ git --version``
+
+Likewise for Ruby installation.
+
+``$ ruby --version``
+
+### Setup
+1. Clone the GitHub repository using the this link: https://github.com/erickhansx/capstone-ruby
+2. Checkout to the ``dev`` branch
+3. Run ``$ bundle install`` in your terminal to install all gems required to run the app.
+
+### Install
+
+Install this project with:
+
+``sh
+  cd my-project
+  gem install
+  ``
+
+
+### Usage
+
+To run the project, execute the following command:
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
+
+### Run tests
+
+To run tests, run the following command:
+
+``sh
+  rspec
+``
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Erick Hans Martinez**
+
+- GitHub: [@erickhansx](https://github.com/erickhansx)
+- Twitter: [@TheErickHans](https://twitter.com/TheErickHans)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+👤 **NDORRH OSWALD Ebu**
+
+- GitHub: [@ndorrh](https://github.com/ndorrh)
+- Twitter: [@Ndorrh Ebu](https://twitter.com/NdorrhEbu)
+- LinkedIn: [ndorrh](https://linkedin.com/in/ndorrhoswaldebu)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Not available]**
+
```

Comment:
I think it is a good practice always to add at least one feature you can add to your project in the future or an existing part in the project that can be improved in the future